app/controllers/items_controller.rb

```diff
@@ -0,0 +1,57 @@
+class ItemsController < ApplicationController
+
+  def index
+    if params[:active] == "true"
+      items = Item.all
+      @items = items.select {|item| item.active == true}
```

Comment:
Rather than using .select here (which is a Ruby method), it would be best to use the ActiveRecord `where` method. It is always more efficient to use ActiveRecord over pure Ruby since databases are more efficient at doing these look ups. You won't notice a difference now, but if you had 10 million items you would. Also, since this is "data" logic, you should make a new model method called something like `active_items`